modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -71,6 +71,13 @@ public JdbcDatabaseContainer(final DockerImageName dockerImageName) {
      */
     public abstract String getJdbcUrl();
 
+    /**
+     * @return a JDBC URL that may be used to connect to the dockerized DB using a custom database name
+     */
+    public String getJdbcUrl(String customDatabaseName) {
```

Comment:
Not `abstract`, so that 3rd part implementations don't need to do anything. Existing code will stay working.